__tests__/app.test.js

```diff
@@ -114,6 +113,46 @@ describe('/api/reviews/:review_id', () => {
                     expect(body.msg).toBe("Bad request");
                 });
         });
+        test("should now show comment count = 0", () => {
+            return request(app)
+                .get("/api/reviews/1")
+                .expect(200)
```

Comment:
This should not be a separate test, this change should be added to the above test and should still pass all the same.